Source/Utils/WarpXUtil.H

```diff
@@ -249,6 +249,8 @@ double parseStringtoReal(std::string str);
  */
 int parseStringtoInt(std::string str, std::string name);
 
+int parseSignalNameToNumber(const std::string &str);
```

Comment:
Little AMReX convention that I wish there was a `clang-format` rule for + Doxygen:
```suggestion
/** Doxygen Title
 *
 * Doxygen brief description. Yep. So many
 * lines.
 *
 * @param str some description
 * @return some other description
 */
int parseSignalNameToNumber (const std::string &str);
```